package.json

```diff
@@ -38,7 +38,6 @@
     "babel-core": "^6.26.3",
     "babel-runtime": "^6.26.0",
     "bnc-onboard": "1.36.0",
```

Comment:
this is also replaced with web3-onboard: https://github.com/blocknative/web3-onboard

Reply:
Updates are still in progress. @FSM1 will jump in for the updates